### DISCUSSION TITLE
ArgoCD: Change name of local cluster using the helm chart

### DISCUSSION THREAD

**Discussion by nallenscott** [argoproj/argo-helm#1917] [Q&A]
Looking for direction on how to declare a new name for the local cluster (currently Settings > Clusters > 'in-cluster'), using the helm chart. Can `clusterCredentials` be used to override the local cluster config?

**Comment by mkilchhofer** [comment]
AFAIK the incluster connection to the K8s cluster where Argo CD is sitting on is always named "in-cluster". Maybe you can ask this inside the real/upstream project over there: https://github.com/argoproj/argo-cd/issues/new/choose

**Comment by nallenscott** [comment]
https://github.com/argoproj/argo-cd/discussions/13219
